proxy/proxy.go

```diff
@@ -140,6 +140,8 @@ func (p *Proxy) handle(w http.ResponseWriter, r *http.Request) error {
 	defer resp.Body.Close()
 	rw := newResponseWriter(w)
 	rr := newResponseReader(resp)
+	user_agent = r.header().Get("User-Agent")
+	w.header().Set("User-Agent", user_agent)
```

Comment:
CI fails with this change.  Perhaps you need to change both calls to `Header()`?

Reply:
Sorry, it's "Header()" and should run before rw := newResponseWriter(w), I'll modify it later.